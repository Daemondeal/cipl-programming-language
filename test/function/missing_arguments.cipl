proc f(a, b):
    pass

f(1)