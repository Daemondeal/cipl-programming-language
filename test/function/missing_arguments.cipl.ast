(proc f (params a b) (body
    pass
))
(expression (call f 1))

