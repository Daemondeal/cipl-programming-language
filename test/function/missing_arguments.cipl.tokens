Token          Lexeme         Line 
PROC           proc           1    
IDENTIFIER     f              1    
LEFT_PAREN     (              1    
IDENTIFIER     a              1    
COMMA          ,              1    
IDENTIFIER     b              1    
RIGHT_PAREN    )              1    
COLON          :              1    
LINE_END       \n             2    
INDENT                        2    
PASS           pass           2    
LINE_END       \n             3    
DEDENT                        4    
IDENTIFIER     f              4    
LEFT_PAREN     (              4    
NUMBER         1              4     1
RIGHT_PAREN    )              4    
LINE_END       \n             4    
EOF                           4    
