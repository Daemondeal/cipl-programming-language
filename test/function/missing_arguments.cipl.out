::stdout::

::stderr::
Error: Expected 2 arguments but got 1 for <proc f>() at line 4
